src/main/java/frc/robot/subsystems/elevator/ElevatorStuff.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot.subsystems.elevator;
+
+import frc.robot.hardware.digitalinput.IDigitalInput;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.robot.hardware.signal.InputSignal;
+
+public record ElevatorStuff(
```

Comment:
create another constructor without the need to enter the digitalinput logpath? (logPath + "digitalInput")
not really needed tho